templates/search/results-en.html

```diff
@@ -22,139 +23,142 @@
 </form>
 
 <section class="results">
```

Comment:
This section should have an id instead of class, since the results is already in a div inside


```suggestion
<section id="wb-land">
```